language-server/modules/langserver-core/src/test/resources/formatting/tableQuery.bal

```diff
@@ -180,4 +180,7 @@ function printTable(string stmt, string tableName, table<anydata> returnedTable)
     } else {
         io:println("Error in table to json conversion");
     }
-}
+
+    table <caa : ApplicationRequest >  result=check  connection -> select ("Select * from Application" ,
+        caa : ApplicationRequest) ;
+}
```

Comment:
Missing a new line at EOF :)

Reply:
it's part of the test ... formatter should add a EOF line

Reply:
this line is available in the expectedTableQuery.bal in `resources/formatting/expected` dir